uat/testing-features/src/main/resources/greengrass/components/recipes/client_java_sdk.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: aws.greengrass.client.Mqtt5JavaSdkClient
+ComponentVersion: '1.0.0'
+ComponentDescription: MQTT5 Java Client powered by AWS Java SDK
+ComponentPublisher: Amazon
+ComponentConfiguration:
+  DefaultConfiguration:
+    deviceId: agent1
+    broker:
+      address: 127.0.0.1
+      port: 47619
+Manifests:
+  - Artifacts:
+      - URI: classpath:/greengrass/artifacts/aws.greengrass.client.Mqtt5JavaSdkClient.jar
+        Permission:
+          Read: ALL
+          Execute: ALL
+    Lifecycle:
+      Run: |
+        java -jar {artifacts:path}/aws.greengrass.client.Mqtt5JavaSdkClient.jar {configuration:/deviceId} {configuration:/broker/address} "{configuration:/broker/port}"
```

Comment:
To avoid missunderstanding rename to agentId. DeviceId means thing name terms of IoT Core, but that is string used only to identify running instance of agent for control.

Reply:
Added new line accd767748763b33cfaa1e179b4c6e220b80b787

Reply:
Could you check comment above, please, and confirm that we should use "agent" in naming?

Reply:
fixed 576c2a16bde5d499ca1b6c7152868cc84aa53013